docs/.vuepress/tools/build.mjs

```diff
@@ -10,6 +10,7 @@ import {
   getVersions,
   FRAMEWORK_SUFFIX
 } from '../helpers.js';
+import { initLog as initSnippetTransformationLog } from './snippet-transform/helpers/previewLogger.js';
```

Comment:
I think it would be more consistent if it was called `initSnippetTransformerLog`